buffer/buffer.mbt

```diff
@@ -148,6 +196,176 @@ pub fn to_bytes(self : T) -> Bytes {
   Bytes::from_fixedarray(self.data, len=self.len)
 }
 
+///| Convert an UInt64 into a 8 bytes long Bytes.
+/// 
+/// # Example
+/// 
+/// ```
+/// let uint64_bytes = 1145141919810 |> @buffer.uint64_to_bytes
+/// ```
+pub fn uint64_to_bytes(i : UInt64) -> Bytes {
+  let byte1 = i.to_byte()
+  let byte2 = (i >> 8).to_byte()
+  let byte3 = (i >> 16).to_byte()
+  let byte4 = (i >> 24).to_byte()
+  let byte5 = (i >> 32).to_byte()
+  let byte6 = (i >> 40).to_byte()
+  let byte7 = (i >> 48).to_byte()
+  let byte8 = (i >> 56).to_byte()
+  [byte1, byte2, byte3, byte4, byte5, byte6, byte7, byte8]
+  |> Bytes::from_fixedarray
+}
+
+///| Convert a 8 bytes long Bytes into an UInt64 **without** a length check.
+/// This function does not check the input so take the risk at your own.
+/// 
+/// # Example
+/// 
+/// ```
+/// let uint64_number = b"\x11\x45\x14\x19\x19\x81\x00\x00" |> @buffer.bytes_to_uint64
+/// ```
+pub fn bytes_to_uint64(bs : Bytes) -> UInt64 {
+  let uint64_1 = bs[0] |> Byte::to_uint |> UInt::to_uint64
+  let uint64_2 = (bs[1] |> Byte::to_uint |> UInt::to_uint64) << 8
+  let uint64_3 = (bs[2] |> Byte::to_uint |> UInt::to_uint64) << 16
+  let uint64_4 = (bs[3] |> Byte::to_uint |> UInt::to_uint64) << 24
+  let uint64_5 = (bs[4] |> Byte::to_uint |> UInt::to_uint64) << 32
+  let uint64_6 = (bs[5] |> Byte::to_uint |> UInt::to_uint64) << 40
+  let uint64_7 = (bs[6] |> Byte::to_uint |> UInt::to_uint64) << 48
+  let uint64_8 = (bs[7] |> Byte::to_uint |> UInt::to_uint64) << 56
+  uint64_1 +
+  uint64_2 +
+  uint64_3 +
+  uint64_4 +
+  uint64_5 +
+  uint64_6 +
+  uint64_7 +
+  uint64_8
+}
+
+///| Convert an UInt into a 4 bytes long Bytes.
+/// 
+/// # Example
+/// 
+/// ```
+/// let uint_bytes = 114514 |> @buffer.uint_to_bytes
+/// ```
+pub fn uint_to_bytes(i : UInt) -> Bytes {
```

Comment:
For the sake of API consistency, I think the naming and location of these conversion functions need further consideration, such as whether bytes_to_xxx can be made to xxx(self: bytes), and then whether xxx_to_bytes should not be written here.

Reply:
Moved xxx_to_bytes into their own type definitions, but for bytes_to_xxx methods I think they should be kept. The bytes_to_xxx methods is only for specific long Bytes rather general Bytes.

Reply:
Removed bytes_to_xxx methods.